server/utils/validation.js

```diff
@@ -0,0 +1,15 @@
+const Joi = require("joi");
+
+const mentionValidation = (data) => {
+  //Validation schema
+  const schema = Joi.object({
+    title: Joi.string().min(6).required(),
+    content: Joi.string().min(10).required(),
+    platform: Joi.string().required(),
+    date: Joi.date().required(),
+    popularity: Joi.optional(),
+    image: Joi.optional(),
```

Comment:
@johnsciutto you'll want to add url validation here as well, should be easy with Joi (uri)

Reply:
On it!